src/core/navigation/Navigation.tsx

```diff
@@ -120,7 +174,24 @@ export function Navigation({
   const logoProps: LogoProps = {
     size: 'large',
     src: LOGO_SOURCES[currentLanguageCode],
-    alt: LOGO_LABELS[currentLanguageCode],
+    alt: HELSINKI_TRANSLATIONS[currentLanguageCode],
+  };
+
+  const localizedCityOfHelsinki =
+    CITY_OF_HELSINKI_TRANSLATIONS[currentLanguageCode];
```

Comment:
when translations are added though the config we dont need this code anymore

Reply:
Moved these to config:
 - LOGO_ARIA_LABELS -> config.translations.cityOfHelsinki
 - LOGO_LABELS -> config.translations.helsinki
 - LOGO_SOURCES -> config.translations.helsinkiLogo